### ISSUE TITLE
ModuleNotFoundError: No module named 'flynt.linting'

### ISSUE THREAD

**Issue by leres** [ikamensh/flynt#57]
Sorry to file another issue so soon but there seems to be a regression since 0.45.4. If I "python setup.py install" 0.45.4 I get a working installation but installing anything newer (including 0.45.5) results in a runtime error:
```
% flynt --verbose *.py
Traceback (most recent call last):
  File "/usr/local/bin/flynt", line 11, in <module>
    load_entry_point('flynt==0.45.5', 'console_scripts', 'flynt')()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.7/site-packages/flynt-0.45.5-py3.7.egg/flynt/__init__.py", line 7, in <module>
  File "/usr/local/lib/python3.7/site-packages/flynt-0.45.5-py3.7.egg/flynt/cli.py", line 6, in <module>
  File "/usr/local/lib/python3.7/site-packages/flynt-0.45.5-py3.7.egg/flynt/api.py", line 12, in <module>
  File "/usr/local/lib/python3.7/site-packages/flynt-0.45.5-py3.7.egg/flynt/process.py", line 11, in <module>
  File "/usr/local/lib/python3.7/site-packages/flynt-0.45.5-py3.7.egg/flynt/transform/transform.py", line 11, in <module>
  File "/usr/local/lib/python3.7/site-packages/flynt-0.45.5-py3.7.egg/flynt/transform/FstringifyTransformer.py", line 7, in <module>
ModuleNotFoundError: No module named 'flynt.linting'
```
I'm running Python 3.7.7 on FreeBSD 11.3-RELEASE-p7 with astor 0.8.1 and tokenize-rt 3.2.0

**Comment by ikamensh** [comment]
Thanks for reporting this fast, it was a bad deployment. Folders were incorrectly specified in the setup.py. Fixed in 0.46

**Comment by leres** [comment]
I actually saw the problem when I initially installed (from master) and assumed I was doing something wrong so I switched to 0.45.4 which worked.
